File: app/src/main/java/com/blogspot/e_kanivets/moneytracker/activity/AboutActivity.java
package com.blogspot.e_kanivets.moneytracker.activity;

import android.text.method.LinkMovementMethod;

import com.blogspot.e_kanivets.moneytracker.activity.base.BaseBackActivity;
import com.blogspot.e_kanivets.moneytracker.databinding.ActivityAboutBinding;

public class AboutActivity extends BaseBackActivity {

    private ActivityAboutBinding binding;

    @Override
    protected void onCreate(android.os.Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        
        binding = ActivityAboutBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        initToolbar();
        initViews();
    }

    private void initViews() {
        binding.tvAbout.setMovementMethod(LinkMovementMethod.getInstance());
    }
}

File: app/src/main/java/com/blogspot/e_kanivets/moneytracker/activity/AboutActivity.kt
package com.blogspot.e_kanivets.moneytracker.activity

import android.text.method.LinkMovementMethod
import android.os.Bundle
import com.blogspot.e_kanivets.moneytracker.activity.base.BaseBackActivity
import com.blogspot.e_kanivets.moneytracker.databinding.ActivityAboutBinding

class AboutActivity : BaseBackActivity() {

    private lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        
        binding = ActivityAboutBinding.inflate(layoutInflater)
        setContentView(binding.root)

        initToolbar()
        initViews()
    }

    private fun initViews() {
        binding.tvAbout.movementMethod = LinkMovementMethod.getInstance()
    }
}
